python/pyarrow/parquet.py

```diff
@@ -560,6 +560,15 @@ def _sanitize_table(table, new_schema, flavor):
     enabled, then dictionary is preferred.
     The byte_stream_split encoding is valid only for floating-point data types
     and should be combined with a compression codec.
+col_encoding : dict, default None
```

Comment:
I was thinking we can maybe also use the full `column_encoding` ? (that's only a bit longer)